signing/transaction-manager.js

```diff
@@ -1,12 +1,39 @@
 import BigNumber from 'bignumber.js'
 import { makeStdTx, makeSignDoc } from '@cosmjs/launchpad'
+import { encodeSecp256k1Pubkey } from '@cosmjs/amino'
+import {
+  AminoTypes,
+  defaultRegistryTypes,
+  createBankAminoConverters,
+  createGovAminoConverters,
+  createStakingAminoConverters,
+} from '@cosmjs/stargate'
+import {
+  encodePubkey,
+  makeAuthInfoBytes,
+  Registry,
+} from '@cosmjs/proto-signing'
+import { fromBase64, toBase64 } from '@cosmjs/encoding'
+import { Int53 } from '@cosmjs/math'
+import { SignMode } from 'cosmjs-types/cosmos/tx/signing/v1beta1/signing'
+import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx'
+import { BroadcastMode } from 'cosmjs-types/cosmos/tx/v1beta1/service'
 import axios from 'axios'
 import { getSigner } from './signer'
 import messageCreators from './messages.js'
 import fees from '~/common/fees'
 import network from '~/common/network'
 import { signWithExtension } from '~/common/extension-utils'
 
+const prefix = 'cosmos'
```

Comment:
What is the meaning for this `prefix` ? Message prefix?

Reply:
It's the address prefix, already referenced from network config